pkg/util/paramtable/service_param.go

```diff
@@ -624,9 +630,88 @@ func (k *KafkaConfig) Init(base *BaseTable) {
 	k.ProducerExtraConfig.Init(base.mgr)
 }
 
+// /////////////////////////////////////////////////////////////////////////////
+// --- pebblemq ---
+type PebblemqConfig struct {
+	Enable   ParamItem `refreshable:"false"`
+	Path     ParamItem `refreshable:"false"`
+	PageSize ParamItem `refreshable:"false"`
+	// RetentionTimeInMinutes is the time of retention
+	RetentionTimeInMinutes ParamItem `refreshable:"false"`
+	// RetentionSizeInMB is the size of retention
+	RetentionSizeInMB ParamItem `refreshable:"false"`
+	// CompactionInterval is the Interval we trigger compaction,
+	CompactionInterval ParamItem `refreshable:"false"`
+	// TickerTimeInSeconds is the time of expired check, default 10 minutes
+	TickerTimeInSeconds ParamItem `refreshable:"false"`
+}
+
+func (r *PebblemqConfig) Init(base *BaseTable) {
+	r.Enable = ParamItem{
+		Key:     "pebblemq.enable",
+		Version: "2.0.0",
```

Comment:
version means Which version was this config introduced in

Reply:
The latest version is [milvus-2.2.13](https://github.com/milvus-io/milvus/releases/tag/v2.2.13). Should I use 2.2.14?